framework/src/modules/chain/submodules/loader.js

```diff
@@ -15,6 +15,7 @@
 'use strict';
```

Comment:
I would suggest to clean and move the entire file to a ES6 class while removing callbacks and using async await when needed.

Reply:
Makes sense. I did this for some functions but it's safer if we do this in multiple steps.

Reply:
We can proceed as you desire. Just open consequents issues to convert the file to the ES6 standard and cover with tests.

Reply:
@diego-G #3280